voluptuous/util.py

```diff
@@ -14,7 +16,11 @@ def Lower(v):
     >>> s('HI')
     'hi'
     """
-    return str(v).lower()
+    if _PY2 and isinstance(v, unicode):
```

Comment:
Maybe replace this repeated fragment with a mini helper function? `_fix_str(v)` or something?